nixos/modules/i18n/input-method/fcitx5.nix

```diff
@@ -10,6 +10,12 @@ let
     then pkgs.qt6Packages.fcitx5-with-addons.override { inherit (cfg) addons; }
     else pkgs.libsForQt5.fcitx5-with-addons.override { inherit (cfg) addons; };
   settingsFormat = pkgs.formats.ini { };
+  mkFcitxKeyValue = lib.flip lib.generators.mkKeyValueDefault "=" {
```

Comment:
We don't really need this lib.flip here, right? And maybe we can use `mkKeyValue` instead of `mkFcitxKeyValue` and `inherit` it later.

Reply:
I used lib.flip as I thought it was easier to read, but it isn't necessary. I've made the changes and switched to mkKeyValue isntead.